source/shared/net/guides/become-baker.rst

```diff
@@ -75,7 +75,8 @@ and send a ``baker add`` transaction to the network:
 
 .. code-block:: console
 
-   $concordium-client baker add <keys-file>.json --sender bakerAccount --stake <amount-to-stake> --out <concordium-data-dir>/baker-credentials.json
+   $concordium-client baker add <keys-file>.json --sender bakerAccount --stake <amount-to-stake> --open-delegation-for <none> --delegation-transaction-fee-commission <5.0e-2> --delegation-baking-commission <5.0e-2> --delegation-finalization-commission <1.0> --baker-url <baker-url>.json --keys-in <baker-keys-new>.json --keys-out <baker-creds-new>.json
```

Comment:
It looks like some of the documentation below needs to be updated -- in particular `<concordium-data-dir>` is no longer referenced in the command. It might also be better to drop the `<>` from the command, because probably people will include them and get funny errors.

Reply:
Also, maybe use 0.05 instead of 5.0e-2.